casanovo/casanovo.py

```
@@ -708,6 +707,12 @@ def _setup_output(
         output_root = (
             f"casanovo_{datetime.datetime.now().strftime('%Y%m%d%H%M%S')}"
         )
+    else:
+        root = Path(output_root)
+        if output_root in {"", ".", ".."} or root.name != output_root:
+            raise click.BadParameter(
+                "--output_root must be a file base name without path separators"
+            )
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)